app/src/main/java/org/dosomething/letsdothis/ui/LoginActivity.java

```diff
@@ -72,6 +72,7 @@ public void onEventMainThread(LoginTask task)
     {
         if(AppPrefs.getInstance(this).isLoggedIn())
         {
+            broadcastLogInSuccess(this);
             Toast.makeText(this, "success login", Toast.LENGTH_SHORT).show();
             startActivity(MainActivity.getLaunchIntent(this));
             finish();
```

Comment:
I don't think we'll need this anymore since it should be handled by the broadcast receiver. I'll test.